src/containers/CreateDoc/index.tsx

```diff
@@ -1,26 +1,120 @@
-import React from 'react';
-import Dropdown from '../../components/Dropdown';
-import documentTypes from './config';
-import { createPdf } from './pdf';
+import React, { useState } from "react";
+import Dropdown from "../../components/Dropdown";
+import documentTypes from "./config";
+import { createPdf } from "./pdf";
+import "./createDoc.css";
+
+import Button from "@material-ui/core/Button";
```

Comment:
Keep imports sorted, all the external libs, like react or material, should be on top, after that import the local modules.  : )

Reply:
you kindamissed this comment, not a big deal but still FYI